upstream-community-operators/aws-service/aws-service.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,178 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: aws-service-operator.v0.0.1
+  namespace: placeholder
+  annotations:
+    categories: "OpenShift Optional, Cloud Provider"
+    description: The AWS Service Operator allows you to manage AWS resources using Kubernetes Custom Resource Definitions.
+    containerImage: awsserviceoperator/aws-service-operator:v0.0.1-alpha3
+    createdAt: 2019-02-09T12:00:00Z
+    support: AWS Labs
+spec:
+  displayName: AWS Service Operator
+  description: |-
+    The AWS Service Operator allows you to manage AWS resources using Kubernetes Custom Resource Definitions.
+
+    Using the AWS Service Operator enables a gitops workflow to drive your infrastructure to the desired state leveraging Kubernetes Custom Resource Definitions (CRD), the Kubernetes internal control loop, and AWS CloudFormation orchestration.
+    In order to run the operator the node will need AWS IAM privledges. This can be done through a service like kube2iam. If you are using an Amazon Elastic Container Service for Kubernetes (EKS) cluster, you most likely provisioned this with CloudFormation.
+    The command shown below will use the CloudFormation stacks to try to update the proper Instance Policy with the correct permissions. Make sure to replace ${STACK_NAME} with the nodegroup stack name from the CloudFormation console.
+    
+      aws iam attach-role-policy --policy-arn arn:aws:iam::aws:policy/AdministratorAccess --role-name $(aws cloudformation describe-stacks --stack-name ${STACK_NAME} --output json | jq -r ".Stacks[0].Outputs[0].OutputValue" | sed -e 's/.*\///g')
+    
+    Before running this operator, you will need to create a secret that contains your cluster-name, region, and account-id. You can do so based off the template below
+    apiVersion: v1
+    kind: Secret
+    metadata:
+      name: aws-service-operator-config
+      namespace: <NAMESPACE>
+    type: Opaque
+    stringData:
+        CLUSTER_NAME: "<CLUSTER_NAME>"
+        REGION: "<REGION>"
+        ACCOUNT_ID: "<000000000000>"
+      
+  keywords: ['AWS']
+  version: 0.0.1
+  maintainers:
+  - name: AWS Labs
+    email: heichris@amazon.com
+  provider:
+    name: AWS Labs
+  labels:
+    name: aws-service-operator
+  selector:
+    matchLabels:
+      name: aws-service-operator
+  links:
+  - name: AWS Operator Source Code
+    url: https://github.com/awslabs/aws-service-operator
+  installModes:
+  - type: OwnNamespace
+    supported: false
+  - type: SingleNamespace
+    supported: false
+  - type: MultiNamespace
+    supported: false
+  - type: AllNamespaces
+    supported: true
```

Comment:
Question: by default this operator will watch all namespaces?

Reply:
The operator seems to watch for CR's in all namespaces independent of which `installMode` is given as a result of the deployment.

Reply:
The operator does not accept a target/watch namespace so it can only watch `AllNamespaces`

Reply:
Did you test it in `OwnNamespace`? I am curious how it can be configured in this mode. Right now it doesn't look like the operator reads any namespace value that can be provided by OLM.